modules/subprocess/src/main/scala/notebook/kernel/pfork/BetterFork.scala

```diff
@@ -170,7 +169,7 @@ object BetterFork {
         } else {
           acc ++ (cl.asInstanceOf[URLClassLoader].getURLs map { u =>
             val f = new File(u.getFile)
-            URLDecoder.decode(f.getAbsolutePath, "UTF8")
+            URLDecoder.decode(f.getParent, "UTF8")
```

Comment:
Hey andy, 

have you had the chance to look into this further? I must admit only skimming it, but I'm interested how this can help.

cya, carsten